libmozdata/vcs_map.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import os
+import sys
+import urllib.request
+
+import hglib
+import requests
+from tqdm import tqdm
+
+MAPPER_SERVICE = "https://mapper.mozilla-releng.net"
+VCS_MAP_FULL_PATH = "vcs_map_full"
+VCS_MAP_CACHE_PATH = "vcs_map_cache"
+
+git_to_mercurial_mapping = {}
+mercurial_to_git_mapping = {}
+
+
+def download_mapfile():
+    if not os.path.exists(VCS_MAP_FULL_PATH):
+        urllib.request.urlretrieve(
+            f"{MAPPER_SERVICE}/gecko-dev/mapfile/full", VCS_MAP_FULL_PATH
+        )
+
+
+MAPFILE_LOADED = False
```

Comment:
I generally prefer creating a class+shared instance, instead of managing globals.

Reply:
Since this is a small module, I prefer to have it like this so you need less boilerplate to use it